types/const.go

```diff
@@ -15,6 +15,8 @@ const (
 	Pip      = "pip"
 	Pipenv   = "pipenv"
 	Poetry   = "poetry"
+	Egg      = "egg"
+	Wheel    = "wheel"
```

Comment:
I think we can aggregate them into `python`.

Reply:
Yes, I too thought that. It its okay with you to merge them both, let me know where do you want to merge at fanal or at go-dep-parser.